networkml/featurizers/README.md

```diff
@@ -0,0 +1,78 @@
+# Features in NetworkML Models
+
```

Comment:
Add note to say this is specific to host.py only since there are other featurizers in this directiory.

Reply:
Done. I changed the title.